### DISCUSSION TITLE
[categorie_corrigee] Simplification de l'algorithme Cas 4 & 7 & 10

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#8] [Ideas]
Les algorithme défini pour le Cas 4 & 7 & 10 dans le chapitre 3.1.11 est très similaire au Cas 1 2 et 3 et peut être simplifié comme suit:

S’il y a des réponses aux questions spécifiques des itinérants(Q25-27), le cycliste sera attribué à la catégorie **"Itinérant"**.
Si le type de sortie déclarée (Q2) est « une randonnée sur plusieurs jours » ET la durée de séjour est supérieure à 1 jour (Q18), le cycliste sera attribué à la catégorie **"Itinérant"**
Dans tous les autres cas, le cycliste sera attribué à la catégorie **[categorie_visuelle_cycliste]**

En fait c'est la même chose que les cas 1 & 2 & 3 avec des catégories visuelles et de questionnaire inversées

**Comment by SteffMangin** [comment]
Là encore avant de valider, je voudrai me remettre dedans. Les vacances ont un effet étrange sur mon cerveau ;-). 
Est-ce que si je fais ça ce midi cela te convient ?

A+

**Reply by SteffMangin** [reply]
C'est OK pour moi, même remarque que pour la traduction des cas 1, 2 et 3.
